### PR TITLE
Carolynhu kubectlconfig

### PR DESCRIPTION
**Pull request by MBishop17**

### CODE REVIEW

File: content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md [hunk 1]
```diff
@@ -3,12 +3,19 @@ title: Using kubectl to Access a Cluster
 weight: 3450
 draft: true
 ---
-Coming Soon
+You can access and manage your Kubernetes clusters using the kubectl shell.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md]: `using the kubectl shell in the UI or configuring it to your local desktop.`

File: content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md [hunk 2]
```diff
@@ -3,12 +3,19 @@ title: Using kubectl to Access a Cluster
...
 
 ## Using kubectl Shell
 
-Coming Soon
+1. From the **Global** view, open the cluster that you want to access with kubectl. 
 
-## Configuring My kubectl
+2. Click **Launch kubectl**.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md]: ```3. A modal will open that connects directly to your Kubernetes cluster. You can start running any `kubectl` commands to interact with the cluster. ```

File: content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md [hunk 3]
```diff
@@ -3,12 +3,19 @@ title: Using kubectl to Access a Cluster
 weight: 3450
 draft: true
 ---
-Coming Soon
+You can access and manage your Kubernetes clusters using the kubectl shell.
 
 ## Using kubectl Shell
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md]: ```## Using Kubectl directly in the UI```

File: content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md [hunk 4]
```diff
@@ -3,12 +3,19 @@ title: Using kubectl to Access a Cluster
...
 
-Coming Soon
+1. From the **Global** view, open the cluster that you want to access with kubectl. 
 
-## Configuring My kubectl
+2. Click **Launch kubectl**.
 
-Coming Soon
+## Configuring My `kubectl`
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md]:
```
## Configuring `kubectl` for your local desktop
```

File: content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md [hunk 5]
```diff
@@ -3,12 +3,19 @@ title: Using kubectl to Access a Cluster
...
+
+After you open kubectl, direct it toward a kubeconfig file to access and manage one of your Kubernetes clusters. To specify a kubeconfig file, click **Kubeconfig File**, copy the contents to an empty file, and then save it to `~/.kube/config`.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/clusters/using-kubectl-to-access-a-cluster/_index.md]: This whole section needs to be re-worked and should be the content from craig's PR on how to download a kubeconfig file.

**Comment by MBishop17** [comment]
@deniseschannon , I made the changes you requested for this PR, but I made them after looking at your comments in Craig's "download kubeconfig" task, so they aren't one-for-one changes.

**Pull request merged**